test/1.4/lib/T_unmapped_access.dml

```diff
@@ -75,64 +75,47 @@ method test() throws {
     /// GREP .*3:0 .*written = 0b1111, previous.*= 0b0000.*
     b.r1_2.write_register(0xffffffff, 0x0000ff0f, NULL);
     b.r1_2.write_register(0xf0f0f0f0, 0xffffffff, NULL);
-    expect(b.r1_2.read_register(0xfffffff00, NULL) == 0xf0f0f000,
-           "default read partially mapped reg");
+    assert b.r1_2.read_register(0xfffffff00, NULL) == 0xf0f0f000;
 
     // segmented by field bit range
     /// GREP \[obj spec-viol\] .*b.r2, bitranges;.*
     /// GREP .*15:8 .*written = 0b11110000, previous.*= 0b00001111.*
     /// GREP .*3:0 .*written = 0b1111, previous.*= 0b0000.*
     b.r2.write_register(0x0f0f0f0f, 0xffffffff, NULL);
-    expect(b.r2.read_register(0xfffffffff, NULL) == 0x0f0ff000,
-           "default read segmented partially mapped reg");
+    assert b.r2.read_register(0xfffffffff, NULL) == 0x0f0ff000;
 
     // Check overridden semantics
     b.r3.write_register(0xfffff0ff, 0xfffffff0, cast(11, void*));
-    expect(most_recent_val == 0xf0ff, "simple custom write_unmapped val");
-    expect(most_recent_mask == 0xfff0, "simple custom write_unmapped mask");
-    expect(most_recent_aux == cast(11, void*), "simple custom write_unmapped aux");
+    assert most_recent_val == 0xf0ff;
+    assert most_recent_mask == 0xfff0;
+    assert most_recent_aux == cast(11, void*);
```

Comment:
oh right this is UB. Hm. Perhaps we should modify the `aux` parameter to be `uintptr_t`.

Reply:
you mean `cast(most_recent_aux, uintptr_t) == 11`?